frontend/src/components/forms/ChooseTemplate/ChooseTemplate.tsx

```diff
@@ -3,20 +3,27 @@
 // we need this to be a client component because nhsuk-react-components uses client-only react features
 
 import { useFormState } from 'react-dom';
-import { chooseTemplateAction } from '@forms/ChooseTemplate/server-action';
 import { NHSNotifyRadioButtonForm } from '@molecules/NHSNotifyRadioButtonForm/NHSNotifyRadioButtonForm';
 import { ZodErrorSummary } from '@molecules/ZodErrorSummary/ZodErrorSummary';
 import { chooseTemplatePageContent } from '@content/content';
-import { FormState } from 'nhs-notify-web-template-management-utils/src/types';
+import {
+  TemplateType,
+  templateTypeDisplayMappings,
+} from 'nhs-notify-web-template-management-utils/src/enum';
+import { chooseTemplateAction } from './server-action';
 
 export const ChooseTemplate = () => {
-  const initialState: FormState = {};
-  const [state, action] = useFormState(chooseTemplateAction, initialState);
+  const [state, action] = useFormState(chooseTemplateAction, {});
+
+  const templateTypes = Object.values(TemplateType);
+  const options = templateTypes.map((templateType) => ({
+    id: templateType,
+    text: templateTypeDisplayMappings(templateType),
```

Comment:
I don't think we should be using the same code that renders the text in the UI for the tests. 

If the code is ever wrong then these tests will still pass. No harm in writing the same function for specifically tests or just hard coding it.

Reply:
Oops. This is wrong my bad. I was looking at the playwright tests and then landed on this somehow.